pugx/geo-form-bundle/0.2/config/packages/pugx_geo_form.yaml

```diff
@@ -0,0 +1,4 @@
+pugx_geo_form:
+    # see https://developers.google.com/maps/documentation/javascript/geocoding#GeocodingRequests for possible values for "region"
+    region: EU
+    useSsl: false
```

Comment:
Shouldn't this be true by default?

Reply:
We always kept it false as default. I guess it's to avoid possible problems with ssl. Anyway I'm not really a maps expert and so I'm open to suggestions

Reply:
At [geocoder-php](https://github.com/geocoder-php/Geocoder) we dropped all non-secure endpoints last year. There is no reason why one should use insecure endpoints. 

I understand why you keep the option but using safe default would be better. 

That is just an opinion, Im happy to merge now if you want to.